packages/react-topology/src/types.ts

```diff
@@ -126,6 +127,7 @@ export interface Node<E extends NodeModel = NodeModel, D = any> extends GraphEle
   setBounds(bounds: Rect): void;
   getPosition(): Point;
   setPosition(location: Point): void;
+  clearPosition(): void;
```

Comment:
I think we can remove `clearPosition`.
All other comments are related to trying to remove this function.